src/current/v23.1/sso-sql.md

```diff
@@ -1,158 +1,265 @@
+
+    The ID of your cluster as specified by the IdP, or a JSON array of such names. This must match `server.oidc_authentication.client_id`; refer to [Single Sign-on (SSO) for DB Console](sso-db-console.html).
+
+    {{site.data.alerts.callout_danger}}
+    Many third-party token issuers, including GCP and Azure, will by default create tokens with a generic default audience. It is best practice to limit the scope of access tokens as much as possible, so if possible, we recommend issuing tokens with only the required audience value corresponding to the `audience` configured on the cluster.
+
+    By extension, if your provider allows you to specify scopes or permissions on the token, you should specify these as restrictively as possible, while still allowing for the functions intended for the service account or user.
+    {{site.data.alerts.end}}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    SET CLUSTER SETTING server.jwt_authentication.audience = '984901724939-njig7lkv7k724rbv2hllvr4of8ul7th7.apps.googleusercontent.com';
+    ~~~
+
+1. `server.jwt_authentication.claim`
+    This setting determines which field in the JWT your cluster will use to the identity to a SQL user. The value of `email` indicates the email field in the JWT.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    SET CLUSTER SETTING server.jwt_authentication.claim = 'email';
+    ~~~
+
+1. `server.jwt_authentication.jwks`
+
+    Add your IdP's public signing key to your cluster's list of accepted signing JSON web keys (JWKS), under the `jwks` setting. This is a [JWK](https://www.rfc-editor.org/rfc/rfc7517) formatted single key or key set, containing the public keys for SSO token issuers/IdPs that will be accepted by your cluster. This list must include a given IdP, or the cluster will reject JWTs issued by it. IdPs serve their public certificates and other required information at `https://{ domain }/.well-known/openid-configuration`.
+    
+    {{site.data.alerts.callout_danger}}
+    IdPs such as Google rotate their signing keys periodically. You must update your cluster with a new signing key before the previous one expires, or your SQL clients will be unable to connect with cluster SSO. We recommend updating this cluster setting with the current key daily to avoid this scenario.
+    {{site.data.alerts.end}}
+
+    {{site.data.alerts.callout_success}}
+    <b>{{ site.data.products.db }} {{ site.data.products.dedicated }} customers:</b> 
+
+    By default, your cluster's configuration will contain the CockroachDB {{ site.data.products.cloud }}'s own public key, allowing CockroachDB {{ site.data.products.cloud }} to serve as an IdP. This is required for [SSO with `ccloud`](https://www.cockroachlabs.com/docs/cockroachcloud/cloud-sso-sql). When modifying this cluster setting, you must include the CockroachDB {{ site.data.products.cloud }} public key in the key set, or SSO with `ccloud` will no longer work.
```

Comment:
I would possibly take this out of the tip box, and just have inline. Or maybe a note. I feel like we want the warning here to really stand out, if know what I mean. I would even bold **We recommend updating this cluster setting with the current key daily to avoid this scenario.** in the warning.